build.sbt

```diff
@@ -19,6 +19,9 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.kinesisConnector,
   Dependencies.Libraries.validator,
   Dependencies.Libraries.nsqClient,
+  Dependencies.Libraries.paranamer,
+  Dependencies.Libraries.jacksonDatabind,
+  Dependencies.Libraries.jacksonCbor,
```

Comment:
`jacksonDatabind` and `jacksonCbor` solved the issue with Kinesis client exception reported by the user and `paranamer` solved the `elastic4s` side.